### DISCUSSION TITLE
Applications of packages to mpox outbreaks

### DISCUSSION THREAD

**Discussion by adamkucharski** [epiverse-trace/epiverse-trace.github.io#284] [Blog post ideas]
Idea from @jamesmbaazam to show how Epiverse tools (and related ones) culd be used in pipelines to support decision-making around novel mpox outbreaks (e.g. like one in South Africa). Think this is really nice idea, and can illustrate what knowledge gaps tools can currently address, and where key gaps for future development are.

A few initial suggestions for examples, based on earlier discussions:

**Understanding transmission**
- If have data on the distribution of cluster sizes, can use {epichains} to estimate R and k. There's also a PR on howto with [an example of how to infer with MCMC](https://github.com/epiverse-trace/howto/pull/57/commits/0fc886a6c4a7b0851c677b730e81d878d3cdd1d7#diff-a3f8033c6d60ebd31aa53d29570ad275f8f49e6c72cf3e09ed3239884289057c).
- If have individual-level transmission chain data, can estimate R and k from the offspring distribution using {superspreading}
- If have data on sexual contacts and secondary attack rate, [can calculate R](https://epiverse-trace.github.io/superspreading/articles/heterogeneous_network_outbreaks.html) accounting for network effects using {superspreading}
- If have aggregated incidence data and want to estimate $R$, and know generation time, can use {EpiNow2} for nowcasting and $R$ estimation, accounting for reporting delays. @sbfnk gave [example here](https://github.com/epiforecasts/nowcasting.example/blob/main/inst/reports/epinow2.md).

**Pipeline infrastructure**
- For line-list data cleaning, could use {cleanepi}, with {numberize} to handle numbers-as-text if needed.
- If want to extract parameters from summary statistics in reports (e.g. range and median) and/or store parameters in easy-to-use objects, can use {epiparameter}

**Exploring scenarios**
- If have R and k estimate, can explore potential transmission chain dynamics, including simple control scenarios, with branching process using {epichains}. Can explore probability of extinction (without need for simulation) under different simple control scenarios with {superspreading}
- If have R and k (or contact distribution and SAR), can simulate outbreaks accounting for some network effects with {simulist}

**Future/wider work (i.e. would need more development)**
- If want to estimate delay distributions, could use {[epidist](https://epidist.epinowcast.org/articles/epidist.html)} then pass resulting estimate to {epiparameter} for use with other packages above.
- If know R and k and delays, could use {ringbp} to simulate contact tracing/quarantine interventions and once ringbp stabilised, {[epinetwork](https://github.com/epiverse-trace/epinetwork/)} to simulate dynamics with interventions over networks using same underlying transmission approach

**Comment by jamesmbaazam** [comment]
Thanks for outlining thoughts @adamkucharski. I had just pushed #285 when you posted this, so you could take a quick look and give comments. It's meant to be a quick/rough draft. I'll update with these points here.

**Comment by Bisaloo** [comment]
This is fixed by https://github.com/epiverse-trace/epiverse-trace.github.io/pull/285 & https://github.com/epiverse-trace/epiverse-trace.github.io/pull/317.

Please feel free to open a new issue if you have ideas for a follow up post.
